src/helpers.js

```diff
@@ -1,5 +1,17 @@
 import MutationObserver from '@sheerun/mutationobserver-shim'
```

Comment:
Simplifying this quite a bit by not using getters and instead just creating variables. Similar to how wait-for-expect does it.